tests/player_helpers.cpp

```diff
@@ -80,3 +87,15 @@ void process_activity( player &dummy )
         }
     } while( dummy.activity );
 }
+
+npc *spawn_npc( const point &p, const std::string &npc_class )
```

Comment:
This function should return a reference instead of a pointer. That would also decrease the changes required in "check_npc_behavior_tree".